src/gui/creds/webflowcredentials.cpp

```diff
@@ -156,8 +156,8 @@ void WebFlowCredentials::askFromUser() {
             _askDialog->setUrl(url);
         }
 
-        QString msg = tr("You have been logged out of %1 as user %2. Please login again")
-                .arg(_account->displayName(), _user);
+        QString msg = tr("You have been logged out of %1 as user %2. Please login again.")
```

Comment:
Minor wording fixes :)
- We don’t need to show "username@cloud.example.com as user username" since that duplicates the username. Just "You have been logged out of cloud.example.com as user username." is enough.
- Second sentence "Please log in again.", "log in" separate.